packages/amplify_authenticator/lib/src/views/confirm_signup_viewmodel.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:amplify_authenticator/src/blocs/auth/auth_bloc.dart';
+import 'package:amplify_authenticator/src/blocs/auth/auth_data.dart';
+import 'package:amplify_authenticator/src/utils/base_viewmodel.dart';
+import 'package:flutter/material.dart';
+
+///Confirm Sign Up View Model
+class ConfirmSignUpViewModel extends BaseViewModel {
+  ///Confirm Sign Up View Model Constructor
+  ConfirmSignUpViewModel(this._authBloc);
+
+  final StateMachineBloc _authBloc;
+
+  final _formKey = GlobalKey<FormState>();
+
+  ///form key
+  GlobalKey<FormState> get formKey => _formKey;
+
+  String _username = "";
```

Comment:
@Jordan-Nelson @dnys1 Do we know if it is better practice to make these nullable instead of empty strings?